gluten-ut/spark34/src/test/scala/org/apache/gluten/utils/velox/VeloxSQLQueryTestSettings.scala

```diff
@@ -240,6 +240,8 @@ object VeloxSQLQueryTestSettings extends SQLQueryTestSettings {
     // -- Aggregate with nulls.
     "group-by.sql",
     "udf/udf-group-by.sql",
+    // Overwrite some results of regr_intercept, regr_r2, corr.
+    "linear-regression.sql",
```

Comment:
the result of these functions is different with vanilla spark?

Reply:
Yes, most statistical functions have differences in precision. We have already fixed these results before, like https://github.com/apache/incubator-gluten/pull/5151.